Merge

justinh998 opened the pull request.

justinh998 committed:
Problem solved that led to an error when the video ID started with a dash

justinh998 committed:
update dependencies

justinh998 committed:
add apt upgrade to have everything up to date

DavidBerdik commented:
Did you intend to close this? It looks to me like it was for the fix that you discussed [here](https://github.com/DavidBerdik/YouTube-Stream-Repeater/issues/4#issuecomment-2038965478)?

justinh998 commented:
I really apologize for the late reply. Yes that was my fix for the problem I described there, the problem is shortly after I fixed it, I noticed that the error I posted in the isseu now always occurs even if the video id does not contain a dash at the beginning even with your version. 
Maybe it is due to some updates of the dependencies (I had deleted the container in between and set it up again.
Could you try if it still works for you, even if you delete the container once and then completely recreate it ?

DavidBerdik commented:
> I really apologize for the late reply.

No problem! Life gets busy sometimes. I get it.

> Could you try if it still works for you, even if you delete the container once and then completely recreate it ?

Sorry, I am a bit confused by what you are asking me to do. Do you want me to test with your pull request or without?
